native/chat/inner-text-message.react.js

```diff
@@ -59,15 +58,9 @@ class InnerTextMessage extends React.PureComponent<Props> {
       messageStyle.backgroundColor = this.props.colors.listChatBubble;
       darkColor = this.props.activeTheme === 'dark';
     }
-    textCustomStyle.height = item.contentHeight;
```

Comment:
I forgot about an important part here that is going to be pretty complex to solve.

To properly layout our chat list, we need to know the height of every single item. This can be difficult, since the height of a chat message depends on the device width and how the text wraps.

Before this PR, we were using `TextHeightMeasurer` to measure the height of the text blob when rendered with the appropriate device width. We need to update that logic to make sure it's measuring the height of the markdown-rendered components.

This will be pretty challenging, so I'll try to handle it.